flows/src/main/java/com/softwaremill/jox/flows/Flow.java

```diff
@@ -588,7 +597,6 @@ case ChannelError(Throwable cause):
                                 if (!buffer.isEmpty()) {
                                     sendBufferAndCleanupCost.call();
                                     // cancel existing timeout and start a new one
-                                    if (timeoutFork != null) timeoutFork.cancelNow();
```

Comment:
is this intentional?

Reply:
Yes, we set timeoutFor=null at 3 lines above, so it would never enter this if. 
In ox it was in `sendBufferAndCleanupCost` method.